pkg/v3/config/config.go

```diff
@@ -63,6 +63,20 @@ type OffchainConfig struct {
 
 	// MaxUpkeepBatchSize is the max upkeep batch size of the OCR2 report.
 	MaxUpkeepBatchSize int `json:"maxUpkeepBatchSize"`
+
+	// LogProviderConfig holds configuration for the log provider
+	LogProviderConfig LogProviderConfig `json:"logProviderConfig"`
+}
+
+type LogProviderConfig struct {
+	// NumOfLogUpkeeps is the number of log upkeeps supported by the registry
```

Comment:
can we expand on this. `NumOfLogUpkeeps` is the number of upkeeps for which we give `FastExecLogsHigh` and `FastExecLogsLow` guarantees. If there are more upkeeps registered then upkeeps will still be performed but the guarantees may get violated

Reply:
cc @amirylm @ferglor please confirm if the above expectation is correct